packages/cli/auth/src/orgs/createOrganizationIfDoesNotExist.ts

```diff
@@ -20,17 +20,62 @@ export async function createOrganizationIfDoesNotExist({
     if (getOrganizationResponse.ok) {
         return false;
     }
-    // if failed response, assume organization does not exist
+
+    if (!getOrganizationResponse.ok) {
+        const error = getOrganizationResponse.error;
+        if (error.error === "UnauthorizedError") {
+            context.failAndThrow(
+                "Failed to check if organization exists: Unauthorized. Please verify your credentials."
+            );
+        } else if (error.error === undefined && error.content != null) {
+            const fetcherError = error.content;
+            if (fetcherError.reason === "timeout") {
+                context.failAndThrow(
+                    "Failed to check if organization exists: Request timed out. Please check your internet connection and try again."
+                );
+            } else if (fetcherError.reason === "unknown") {
+                context.failAndThrow(
+                    `Failed to check if organization exists: Network error. ${fetcherError.errorMessage}\nPlease check your internet connection and try again.`
```

Comment:
and I think we can't definitively say this for unknown, so let's do:
`Failed to check if organization exists: ${fetcherError.errorMessage}`